client/src/legacy/CMSMain.Tree.js

```diff
@@ -140,6 +141,46 @@ $.entwine('ss.tree', function($) {
         }
       };
       return config;
+    },
+
+    canMove: async function(data) {
+      // only display warning if its the homepage
+      var isHomePage = data.rslt.o.find(".homepage").first().length > 0;
+      if (!isHomePage) {
+        return true;
+      }
+
+      // only display warning if we're moving to a new parent
+      var oldParentId = data.rslt.op.data('id');
+      var newParentId = data.rslt.np.data('id');
+      if (oldParentId === newParentId) {
+        return true;
+      }
+
+      var message = i18n.sprintf(
+        i18n._t(
+          'CMS.RemoveHomePageWarningMessage',
+          'Warning: This page has the URL of %s. ' +
+          'This is required by your website to  identify the home page. ' +
+          'Without this, visitors will not be able to view your home page.'
+        ),
+        "<em>home</em>"
+      );
+
+      const regex = /(.*)(<em>home<\/em>)(.*)/;
+      var match = regex.exec(message);
```

Comment:
Looks like you're injecting `home` into the translated string? You can do this using `i18n.inject`. Here's an example: https://github.com/dnadesign/silverstripe-elemental/blob/4/client/src/components/ElementActions/UnpublishAction.js#L32

Reply:
Here I am finding the `home` tag and splitting by it, then replacing it with a tag instead of a string and passing that as an array so that React allows the HTML formatting to stay intact.

I'll add a remark to make this more obvious.